docs/source/devguides/walkthroughs/SingleMachine-Walkthrough.rst

```diff
@@ -0,0 +1,162 @@
+.. _SingleMachine-Walkthrough:
+
+Single Machine Deployment
+=========================
+
+The purpose of this demonstration is to show the process of setting up a simple VOLTTRON instance for use on a single machine.
+
+Install and Build VOLTTRON
+--------------------------
+
+First, :ref:`install <VOLTTRON-Prerequisites>` and :ref:`build <Building-VOLTTRON>` VOLTTRON:
+
+For a quick reference: 
+
+.. code-block:: console
+        
+        sudo apt-get update
+        sudo apt-get install build-essential python-dev openssl libssl-dev libevent-dev git
+        git clone https://github.com/VOLTTRON/volttron/
+        cd volttron
+        python2.7 bootstrap.py
+
+
+Configuring VOLTTRON
+--------------------
+
+After the build is complete, activate the VOLTTRON environment.
+
+.. code-block:: console
+
+        source env/bin/activate
+
+The ``volttron-cfg`` command allows for an easy configuration of the VOLTTRON environment.
+
+.. note::
+        
+        To create a simple instance of VOLTTRON, leave the default response, or select yes (y) if prompted for a yes or no response [Y/N]. You must choose a username and password for the VOLTTRON Central admin account.
+
+A set of example responses are included here:
+
+.. code-block:: console
```

Comment:
There are some missing prompts that happen in the rabbitmq-branch please update with those here.

Reply:
You also might mention that these must be done in an activated environment. eg source env/bin/activate